src/network/tls.py

```diff
@@ -7,18 +10,29 @@
 import ssl
 import sys
 
-from debug import logger
-from network.advanceddispatcher import AdvancedDispatcher
 import network.asyncore_pollchoose as asyncore
-from queues import receiveDataQueue
 import paths
 import protocol
+from debug import logger
+from network.advanceddispatcher import AdvancedDispatcher
```

Comment:
`from advanceddispatcher import AdvancedDispatcher`? It's already inside the `network` module.